src/FluentValidation.Tests/CascadingFailuresTester.cs

```diff
@@ -24,342 +24,479 @@ namespace FluentValidation.Tests {
 	using Xunit;
 
 	public class CascadingFailuresTester : IDisposable {

```

Comment:
This is actually the new test class, with the same name as the old one (see commit comments)